### DISCUSSION TITLE
vhd files

### DISCUSSION THREAD

**Discussion by BobStadler** [copy/v86#1153] [Q&A] [answered]
does it support vhd files instead of the img files for the hdd?

**Comment by copy** [comment, marked as answer]
No (but you can convert vhd to img using qemu-img)

**Reply by BobStadler** [reply]
i dont have a windows or linux pc available,
is there anny other way i could convert it (some online tool or something)
i only have a chromebook, wirh i can not put in developer mode and i dont have android accses.
only a pwa would maybe work
